### PR TITLE
Added a method to save byte[] content, rather than only String content.

### PR DESCRIPTION
**Pull request by jkwatson**

### CODE REVIEW

**Comment by jkwatson** [comment]
helloooooo?  anyone home over there?

File: src/main/java/com/flightstats/filesystem/FileSystem.java [hunk 1]
```diff
@@ -20,6 +20,8 @@
 
     void saveContent(String content, Path fileName, String contentType);
 
+    void saveContent(byte[] content, Path fileName, String contentType);
```
Comment (breedx2) [inline src/main/java/com/flightstats/filesystem/FileSystem.java]: I think all 3 `saveContent` calls could/should flow top-to-bottom into a single one that does the work.  Right now it's a little split and hard to follow.

**Comment by breedx2** [comment]
Just that one comment, rest looks good.

File: src/main/java/com/flightstats/filesystem/FileSystem.java (hunk 1 above)
Comment (jkwatson) [inline src/main/java/com/flightstats/filesystem/FileSystem.java]: I'm confused.  Isn't that what it's doing now?  How would you re-order?

Comment (breedx2) [inline src/main/java/com/flightstats/filesystem/FileSystem.java]: Sorry, I meant in the LocalFileSystem implementation, not in the interface.

Comment (jkwatson) [inline src/main/java/com/flightstats/filesystem/FileSystem.java]: aha! I'll fix that one up.

**Comment by jkwatson** [comment]
Fixed, based on comments.

**Comment by breedx2** [comment]
Let it rip!
